JS-condition/js-conditions-and-booleans_teenager-check/js/logType.js

```diff
@@ -0,0 +1,50 @@
+let data;
+
+let typeOfData = typeof data;
+
+let message = "";
+
+switch (typeOfData) {
+  case "undefined":
+    message = "Undefine!";
+    break;
+
+  case "string":
+    message = "String!";
+    break;
+  case "bigInt":
+    message = "BigInt!";
+    break;
+  case "function":
+    message = "Function!";
+    break;
+  case "boolean":
+    message = "boolean!";
+    break;
+
+  //number & NaN
+
+  case "number":
+    if (Number.isNaN(data)) {
```

Comment:
I don't _think_ you need to specify the `Number` class to use the `isNaN` method.

Reply:
I did this because NaN & number both has type number

Reply:
No, I meant something else but then I did a bit of research and as it turns out, you can ignore this comment 😌